website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+    ```
+
+## Create Peer CRDs
+
+1. Create Peer CRDs for each peer:
+
+    *Client cluster*:
+
+    ```sh
+    echo "
+    apiVersion: clusterlink.net/v1alpha1
+    kind: Peer
+    metadata:
+        name: server
+        namespace: clusterlink-system
+    spec:
+        gateways:
+            - host: server-peer-clusterlink.clusterlink-system.svc.cluster.local
+              port: 6002
+    " | kubectl apply -f -
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    echo "
+    apiVersion: clusterlink.net/v1alpha1
+    kind: Peer
+    metadata:
+        name: client
+        namespace: clusterlink-system
+    spec:
+        gateways:
+            - host: client-peer-clusterlink.clusterlink-system.svc.cluster.local
+              port: 6001
+    " | kubectl apply -f -
+    ```
+
+    To verify that the connectivity between the peers is established correctly,
+    please check if the condition `PeerReachable` has been added to the peer CR status in each cluster.
+
+    ```sh
+    kubectl describe peers.clusterlink.net -A
+    ```
+
+    {{% expand summary="Sample output" %}}
+
+    ```
+    Name:         client
+    Namespace:    clusterlink-system
+    Labels:       <none>
+    Annotations:  <none>
+    API Version:  clusterlink.net/v1alpha1
+    Kind:         Peer
+    Metadata:
+        Creation Timestamp:  2024-05-28T12:47:33Z
+        Generation:          1
+        Resource Version:    807
+        UID:                 1fdeafff-707a-43e2-bb3a-826f003a42ed
+    Spec:
+        Gateways:
+        Host:  172.18.0.4
+        Port:  30443
+    Status:
+        Conditions:
+        Last Transition Time:  2024-05-28T12:47:33Z
+        Message:
+        Reason:                Heartbeat
+        Status:                True
+        Type:                  PeerReachable
+    ```
+
+    {{% /expand %}}
+
+## Connect Application Services
+
+After creating the peer connectivity using FRP, continue to the next step of exporting services, importing services, and creating policies as described in the tutorials [Nginx tutorial][] and [iPerf3 tutorial][].
+
+## Cleanup
+
+To remove all FRP components:
+
+1. Delete FRP server deployment, config-map and ingress service :
+
+    ```sh
+    kubectl delete deployments -n clusterlink-system frps
+    kubectl delete services -n clusterlink-system clusterlink-frps
+    kubectl delete configmaps -n clusterlink-system frps-config
+
+1. Delete FRP client deployment, config-map and ingress service  on each cluster:
+
+    *Client cluster*:
+
+    ```sh
+    kubectl delete deployments -n clusterlink-system frpc
+    kubectl delete services -n clusterlink-system server-peer-clusterlink
+    kubectl delete configmaps -n clusterlink-system frpc-config
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    kubectl delete deployments -n clusterlink-system frpc
+    kubectl delete services -n clusterlink-system client-peer-clusterlink
+    kubectl delete configmaps -n clusterlink-system frpc-config
+    ```
```

Comment:
Since the install did not include the test applications, it's fine that cleanup also excludes it.
Suggest adding a sentence here, and in install section, that needs to be done by the user